meet-data/nzpf/1220/entries.csv

```diff
@@ -2,16 +2,16 @@ Name,Division,WeightClassKg,BestSquatKg,BestBenchKg,BestDeadliftKg,TotalKg,Place
 Mike Gardiner,Masters 1,66,235,140,230,605,1,Single-ply,SBD,M
 Brian Choi,Juniors,66,187.5,125,206,518.5,2,Single-ply,SBD,M
 Bruce Park,Masters 4,66,135,80,187.5,402.5,3,Single-ply,SBD,M
-Ryan Stewart,Special Olympic Athlete,66,55,62.5,95,212.5,4,Single-ply,SBD,M
+Ryan Stewart,Special Olympics,66,55,62.5,95,212.5,4,Single-ply,SBD,M
 John Strachan,Open,74,240,163.5,232.5,636,1,Single-ply,SBD,M
 Steve Barnes,Masters 2,74,222.5,160,240,622.5,2,Single-ply,SBD,M
 Stephen Pickens,Masters 2,74,190,122.5,200,512.5,3,Single-ply,SBD,M
 Jamie King,Juniors,74,170,95,245,510,4,Single-ply,SBD,M
 Harry de Lacey,Juniors,74,185,102.5,212.5,500,5,Single-ply,SBD,M
 Rajah Singh,Open,74,140,102.5,185,427.5,6,Single-ply,SBD,M
-Ming Ming Edgar,Blind Athlete (Juniors),74,140,100,160,400,7,Single-ply,SBD,M
-Joshua Cooke,Special Olympic Athlete (Sub-Juniors),74,100,70,115,285,8,Single-ply,SBD,M
-Bradley Caffell,Special Olympic Athlete (Sub-Juniors),74,82.5,45,87.5,215,9,Single-ply,SBD,M
+Ming Ming Edgar,Blind,74,140,100,160,400,7,Single-ply,SBD,M
+Joshua Cooke,Special Olympics,74,100,70,115,285,8,Single-ply,SBD,M
+Bradley Caffell,Special Olympics,74,82.5,45,87.5,215,9,Single-ply,SBD,M
```

Comment:
This change is actually losing information, though... before, we could maybe include them in the Sub-Juniors and Juniors rankings, but now we'd have no idea.

Reply:
Good point! Will add Sub-Junior, Junior subdivisions for Special Olympics/Blind.